python-backend/app/api/users/core/models/core_user.py

```diff
@@ -0,0 +1,73 @@
+import uuid
+from datetime import datetime
+from flask_restplus import fields
+
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.orm import validates, backref
+from sqlalchemy.schema import FetchedValue
+from app.extensions import db, api
+
+from .idir_user_detail import IdirUserDetail
+from .idir_membership import IdirMembership
+from .idir_membership_xref import IdirMembershipXref
+
+from ....utils.models_mixins import AuditMixin, Base
+
+
+class CoreUser(AuditMixin, Base):
+    __tablename__ = 'core_user'
+    core_user_id = db.Column(db.Integer, primary_key=True, server_default=FetchedValue())
+    core_user_guid = db.Column(UUID(as_uuid=True), nullable=False, server_default=FetchedValue())
+    email = db.Column(db.String(254))
+    phone_no = db.Column(db.String)
```

Comment:
other way around probably, shouldn't have size at all unless we want to go to code first....

Reply:
Hmm.. okay. Maybe I misremembered, but I thought you had asked me to add the sizes to a PR a while back. Or just things have changed since then.

I'll update my current PR to remove the string sizes as well. Your `IdirUserDetail` model has a few sizes that can be removed

Reply:
The size is only used if SqlAlchemy generates the table from code first... which isn't in the roadmap. so it just causes ambiguity if it's written in a place that doesn't do anything.